src/main/java/bio/terra/cli/command/cromwell/generate-config.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Initialize a default Cromwell config. Don't overwrite in case the user has
+# customized their Cromwell config file on their PD.
+echo "cromwell_config" : $1
+echo "GOOGLE_PROJECT": $2
+echo "WORKSPACE_BUCKET": $3
+echo "PET_SA_EMAIL": $4
+if [ ! -f "$1" ]; then
+  cat <<EOF | tee "$1"
+
+google {
+  application-name = "cromwell"
+  auths = [{
+    name = "application_default"
+    scheme = "application_default"
+  }]
+}
+
+backend {
+  default = "PAPIv2-beta"
+  providers {
+
+    # Disables the Local backend
+    Local.config.root = "/dev/null"
+
+    PAPIv2-beta {
+      actor-factory = "cromwell.backend.google.pipelines.v2beta.PipelinesApiLifecycleActorFactory"
+
+      config {
+        project = "$2"
+        concurrent-job-limit = 10
+        root = "$3/workflows/cromwell-executions"
```

Comment:
let's just hard code it here since we are not passing in a real bucket path. 

but also add a comment saying: replace {WORKSPACE_BUCKET} with the path to a gcs bucket in this workspace, e.g. gs://my-cromwell-workflow-bucket/workflows/cromwell-executions.